services/continuousquery/service_test.go

```diff
@@ -85,19 +85,20 @@ func TestService_handle(t *testing.T) {
 	s.handle()
 }
 
-func TestService_hasContinuousQuery(t *testing.T) {
+func TestService_getContinuousQueries(t *testing.T) {
 	s := NewTestService(t)
 	s.MetaClient = &MockMetaClient{
 		DatabasesFn: func() map[string]*meta.DatabaseInfo {
 			return nil
 		},
 	}
 	// test no databases exist
-	if s.hasContinuousQuery() {
-		t.Error("Expected hasContinuousQuery to return false")
+	s.appendContinuousQueries()
+	if len(s.ContinuousQuery) != 0 {
+		t.Error("Expected appendContinuousQueries to return empty")
```

Comment:
try to use:
"github.com/stretchr/testify/assert"
"github.com/stretchr/testify/require"